TemplateViewController.swift

```diff
@@ -25,22 +30,15 @@ class TemplateViewController: UICollectionViewController {
     override func awakeFromNib() {
         super.awakeFromNib()
         dataSource.refresh()
+        collectionView!.backgroundColor = UIColor.backgroundColor
+        collectionView!.layer.borderColor = UIColor.darkAmberColor.CGColor
+        collectionView!.layer.borderWidth = 1
     }
     
     override func prefersStatusBarHidden() -> Bool {
         return true
     }
 
-    override func viewWillAppear(animated: Bool) {
-        super.viewWillAppear(animated)
-        if let layout = collectionViewLayout as? UICollectionViewFlowLayout {
-            let cellWidth = (view.frame.width - 20)/2
-            let screenBounds = UIScreen.mainScreen().bounds
-            let ratio = screenBounds.height/screenBounds.width
-            layout.itemSize = CGSize(width: cellWidth, height: cellWidth*ratio)
-            
-        }
-    }
```

Comment:
Prima dat dit weg is, maar de breedte was niet voor niets afhankelijk van de width...